core/base-service/got.js

```diff
@@ -22,6 +27,12 @@ async function sendRequest(gotWrapper, url, options) {
         underlyingError: new Error('Maximum response size exceeded'),
       })
     }
+    if (err.code in customExceptions) {
+      throw new Inaccessible({
+        ...customExceptions[err.code],
+        underlyingError: err,
+      })
```

Comment:
Spreading in this order means you can't manually override `underlyingError`. I don't think it would ever make sense to do that.